exporter/exporter.go

```diff
@@ -26,13 +28,13 @@ resource "lightstep_metric_dashboard" "exported_dashboard" {
       hidden              = {{.Hidden}}
 {{if (and .SpansQuery .SpansQuery.Query) }}
       spans {
-         query         = "{{escapeSpanQuery .SpansQuery.Query}}"
+         query         = "{{escapeHCLString .SpansQuery.Query}}"
          operator      = "{{.SpansQuery.Operator}}"
          group_by_keys = [{{range .SpansQuery.GroupByKeys}}"{{.}}",{{end}}]{{if eq .SpansQuery.Operator "latency"}}
          latency_percentiles = [{{range .SpansQuery.LatencyPercentiles}}{{.}},{{end}}]{{end}}
       }
 {{end}}{{if .TQLQuery}}
-      tql                 = "{{.TQLQuery}}"
+      tql                 = "{{escapeQueryString .TQLQuery}}"
```

Comment:
wont these quotes get in the way if we're using the heredoc syntax?

Reply:
on the other hand, if we decide to never output `lightstep_metric_dashboard`s when using TQL queries with this tool, then i think we'll never actually need this change?